.travis.yml

```diff
@@ -41,7 +41,7 @@ before_install:
 install: /bin/true
 
 before_script:
-  - git remote set-url origin "https://${GH_TOKEN}@github.com/openzipkin/zipkin.git"
+  - git remote set-url origin "https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}/zipkin.git"
```

Comment:
TRAVIS_REPO_SLUG also contains the repo name, so this will evaluate to github.com/openzipkin/zipkin/zipkin.git